pkg/kubectl/cmd/cp.go

```diff
@@ -195,6 +195,11 @@ func makeTar(filepath string, writer io.Writer) error {
 	// TODO: use compression here?
 	tarWriter := tar.NewWriter(writer)
 	defer tarWriter.Close()
+
+	len := len(filepath)
+	if filepath[len-1] == '/' {
+		filepath = filepath[0 : len-1]
+	}
```

Comment:
Add a test case to prove this?

Reply:
to prove it goes  correctly when the input ends up with '/' ?
by the way,thanks for the reply

Reply:
yeah.

Reply:
Yeap, a unit test for this would be better.